### PR TITLE
Update lsblk commands to make it easier to identify devices.

### LINKED ISSUE #984: Suggest a more detailed lsblk command
**Describe the error/improvement to an existing document or image**
As given in the example in this section:
https://github.com/clearlinux/clear-linux-documentation/blob/master/source/tutorials/multi-boot/dual-boot-win.rst#advanced-use-systemd-boot-to-boot-windows-10-os-and-cl
`lsblk` doesn't give enough detail to identify which partition is which. I suggest adding more columns to its output to help users find the correct partitions, maybe something like:
`lsblk -o NAME,MAJ:MIN,RM,SIZE,RO,TYPE,LABEL,PARTLABEL,PARTTYPE,PARTUUID,UUID,MOUNTPOINT`

PARTTYPE, PARTUUID, and UUID are probably excessive in most cases, but can help narrow it down more in case particularly LABEL and PARTLABEL are insufficient.

### PR DESCRIPTION
**Pull request by bktan8**
Closes #984

Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/get-started/bootable-usb.rst [hunk 1]
```diff
@@ -39,64 +39,53 @@ Burn the |CL| image onto a USB drive
...
+      sudo umount /dev/sdc*
 
-#. Burn the image onto the USB drive. This example burns an image onto `/dev/sdd`.  The device name of the USB may vary.
+#. Burn the image onto the USB drive. This example burns an image onto ``/dev/sdc``.  
+   The device name of the USB may vary.
 
    .. code-block:: bash
 
-      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/sdd oflag=sync bs=4M status=progress
+      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/sdc oflag=sync bs=4M status=progress
```
Comment (mvincerx) [inline source/get-started/bootable-usb.rst]: See Ln 182 (revised) below.  Currently, the Rufus instruction is missing one step. 
Replace 182 with: 
#. Under boot selection , click the :guilabel:`SELECT` button.
#. In the dialog, navigate to where the |CL| iso image was downloadedand select it.

**Comment by mvincerx** [comment]
LGTM. 
:sparkles: Congrats! ✨ You posted PR 1000 on the Clear Linux OS Docs repo. 👍

**Comment by bktan8** [comment]
Sweet!  What's the prize?

**Comment by mvincerx** [comment]
We should decide this in the next docs meeting. Collectively. Open source style.

**Pull request merged**